frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx

```diff
@@ -94,7 +94,7 @@ export function SessionRecordingsFilters({
                     className="w-full rounded-none border-0 border-t"
                     multiple
                     defaultActiveKeys={initiallyOpen ? ['advanced-filters'] : []}
-                    size="small"
+                    size="xsmall"
```

Comment:
?

Reply:
Weird hangover from something else I was experimenting with. Removing in next commit